hubblestack_nova_profiles/cis/windows-2016-level-1-scored-v1-0-0.yaml

```diff
@@ -2334,8 +2334,8 @@ win_reg:
               tag: CIS-19.7.25.1
               match_output: '1'
               value_type: 'equal'
-      description: (l1) ensure 'prevent users from sharing files within their profile.' is set to 'enabled'
-     Do not suggest third-party content in Windows spotlight:
+      description: (l1) ensure 'prevent users from sharing files within their profile' is set to 'enabled'
+    Do not suggest third-party content in Windows spotlight :
```

Comment:
That space doesn't matter.  I put it there to conform with all the other items, but I can just as easily delete all the spaces before the colon as well.  Do you want me to do that?